packages/cli/bin/cli.js

```diff
@@ -145,20 +165,22 @@ async function main() {
         printWarnings(warnings);
         process.exit(1);
       }
-      console.log(`  ${FG_GREEN}✔${RESET}  no errors`);
+      console.log(`  ${FG_GREEN}✔${RESET}  no errors ${time(start)}`);
       break;
     }
     case 'init': {
       if (fs.existsSync(config.tokens)) throw new Error(`${config.tokens} already exists`);
       fs.cpSync(new URL('../tokens-example.json', import.meta.url), new URL(config.tokens, cwd));
-      console.log(`  ${FG_GREEN}✔${RESET} ${config.tokens} created`);
+      console.log(`  ${FG_GREEN}✔${RESET} ${config.tokens} created ${time(start)}`);
       break;
     }
-    default:
+    default: {
       showHelp();
+      break;
+    }
   }
 
-  console.info(`  Done  ${time(start)}`);
```

Comment:
Rolls time output into previous console line. Now there’s not a `Done` line (which was a bit extraneous)